TeleSharp.TL/TLContext.cs

```diff
@@ -14,13 +14,20 @@ public static class TLContext
 
         public static void Init()
         {
-            Types = new Dictionary<int, Type>();
-            Types = (from t in Assembly.GetExecutingAssembly().GetTypes()
-                     where t.IsClass && t.Namespace.StartsWith("TeleSharp.TL")
-                     where t.IsSubclassOf(typeof(TLObject))
-                     where t.GetCustomAttribute(typeof(TLObjectAttribute)) != null
-                     select t).ToDictionary(x => ((TLObjectAttribute)x.GetCustomAttribute(typeof(TLObjectAttribute))).Constructor, x => x);
-            Types.Add(481674261, typeof(TLVector<>));
+            try
+            {
+                Types = new Dictionary<int, Type>();
+                Types = (from t in Assembly.GetExecutingAssembly().GetTypes()
+                         where t.IsClass && t.Namespace.StartsWith("TeleSharp.TL")
+                         where t.IsSubclassOf(typeof(TLObject))
+                         where t.GetCustomAttribute(typeof(TLObjectAttribute)) != null
+                         select t).ToDictionary(x => ((TLObjectAttribute)x.GetCustomAttribute(typeof(TLObjectAttribute))).Constructor, x => x);
+                Types.Add(481674261, typeof(TLVector<>));
+            }
+            catch
```

Comment:
What exception did you receive here? You should add the exception type, as putting "catch-all" blocks like this is quite dangerous.

Reply:
Sometime the error that @Jarzamendia describes top

> I recive An item with the same key has already been added. Key: 481674261 when I use this dll's. :(

This happens if the assembly is loaded twice e.g. from any dependency injection

Reply:
A global catch is never OK (the only exception is in the entry point of your application, for logging and bug-auto-submission purposes). When you're dealing with certain exception of type X, you need to use that type X in the code because the future developers need to know what was happening, what were you trying to fix. Please add the type.

Reply:
Add proper method to prevent double adding type